tests/parquet_test.py

```diff
@@ -14,13 +14,15 @@ def read_write_test(dtype):
         ak_arr = ak.randint(0, 2**32, SIZE)
     elif dtype =='uint64':
         ak_arr = ak.randint(0, 2**32, SIZE, dtype=ak.uint64)
+    elif dtype =='bool':
+        ak_arr = ak.randint(0, 2**32, SIZE, dtype=ak.bool)
         
     ak_arr.save_parquet("pq_testcorrect", "my-dset")
     pq_arr = ak.read_parquet("pq_testcorrect*", "my-dset")
     
     for f in glob.glob('pq_test*'):
         os.remove(f)
-
+        
```

Comment:
Just whitespace:
This is extremely nitpicky and you shouldn't bother to change it unless there's another reason to update the PR but I wanted to make a note that it looks like this whitespace was introduced